### PR TITLE
Feature/Edit course details

### PR DESCRIPTION
**Pull request by Revasree7780**

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Murutechi) [inline .classpath:1]: Remove .classpath file from feature branch

File: .project [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Murutechi) [inline .project:1]: Remove .project folder from feature branch

File: .settings/org.eclipse.core.resources.prefs [hunk 3]
```diff
@@ -0,0 +1,4 @@
+eclipse.preferences.version=1
```
Comment (Murutechi) [inline .settings/org.eclipse.core.resources.prefs:1]: Remove .settings file from feature branch

File: src/main/java/com/jm/lms/studentms/model/EditCourse.java [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "EDITCOURSE")
```
Comment (Murutechi) [inline src/main/java/com/jm/lms/studentms/model/EditCourse.java:13]: Change table name to course_details

File: target/classes/META-INF/MANIFEST.MF [hunk 5]
```diff
@@ -0,0 +1,6 @@
+Manifest-Version: 1.0
```
Comment (Murutechi) [inline target/classes/META-INF/MANIFEST.MF:1]: Remove target folder from feature branch

**Review by Murutechi — CHANGES_REQUESTED** [review]

File: .classpath (hunk 1 above)
Comment (Revasree7780) [reply .classpath]: Done sir

File: .project (hunk 2 above)
Comment (Revasree7780) [reply .project]: Done

File: .settings/org.eclipse.core.resources.prefs (hunk 3 above)
Comment (Revasree7780) [reply .settings/org.eclipse.core.resources.prefs]: Done sir

File: src/main/java/com/jm/lms/studentms/model/EditCourse.java (hunk 4 above)
Comment (Revasree7780) [reply src/main/java/com/jm/lms/studentms/model/EditCourse.java:13]: Changed the table name to course_details

File: target/classes/META-INF/MANIFEST.MF (hunk 5 above)
Comment (Revasree7780) [reply target/classes/META-INF/MANIFEST.MF]: Done

**Comment by Revasree7780** [comment]
all checks passed sir.......

**Pull request closed without merging**